### PR TITLE
OD-356 [Fix] Users will be able to work with accessibility

### PR DESCRIPTION
**Pull request by YaroslavOvdii**
@sofiiakvasnevska 

## Issue
https://weboo.atlassian.net/browse/OD-356

## Description
Fixed accessibility issues from the table.

## Screenshots/screencasts
https://share.getcloudapp.com/RBu4BlWG

## Backward compatibility
This change is fully backward compatible.

## Reviewers
@upplabs-alex-levchenko @ivandevupp @AndrRyaz

### CODE REVIEW

File: js/menu.js [hunk 1]
```diff
@@ -59,22 +59,28 @@ function init() {
       return;
     }
 
-    $('body').toggleClass('disableScroll circle-menu-active');
-    $(this).find('.hamburger').toggleClass('is-active');
-    $(this).toggleClass('active');
-    $('.fl-menu-circle-nav-holder .nav-circle').toggleClass('active');
-    $('.fl-menu-circle-nav-list-holder').toggleClass('active').scrollTop(0);
+    var $this = $(this);
```
Comment (tonytlwu) [inline js/menu.js]: Give `this` or `$this` a name where possible.

```suggestion
    var $toggle = $(this);
```

File: css/menu.css [hunk 2]
```diff
@@ -252,3 +252,7 @@ body.disableScroll {
 [data-fl-exit-app] .fa-angle-right.linked-icon {
   display: none;
 }
+
+.hidden {
```
Comment (tonytlwu) [inline css/menu.css]: Why is this needed?

image

Bootstrap already has this exact same style.

File: build.html [hunk 3]
```diff
@@ -22,7 +22,7 @@
 {{/if}}
 
 <!-- MENU -->
-<div class="fl-menu-body fl-menu-circle-nav-list-holder {{#if location}}fl-menu-circle-left{{/if}}">
+<div class="fl-menu-body fl-menu-circle-nav-list-holder {{#if location}}fl-menu-circle-left{{/if}} hidden">
```
Comment (tonytlwu) [inline build.html:25]: Same concern as https://github.com/Fliplet/fliplet-menu-default/pull/25#discussion_r583045668

**Review by tonytlwu — CHANGES_REQUESTED** [review]

File: build.html (hunk 3 above)
Comment (YaroslavOvdii) [reply build.html:25]: @tonytlwu in this menu transition is working because we use [setTimeout](https://github.com/Fliplet/fliplet-menu-circle/pull/24/files#diff-82df7b14712db47ddbc76d0b8bc9ff3977cc8e1c2217029413661176afc37f97R69) here to give time for the component to remove the `display: none`
[screencast](https://share.getcloudapp.com/7Kupj9ER)

Comment (tonytlwu) [reply build.html:25]: Makes sense. Thanks for the explanation and screencast.

File: css/menu.css (hunk 2 above)
Comment (YaroslavOvdii) [reply css/menu.css]: @tonytlwu removed unneeded class

File: js/menu.js (hunk 1 above)
Comment (YaroslavOvdii) [reply js/menu.js]: @tonytlwu renamed

**Review by tonytlwu — APPROVED** [review]
@sofiiakvasnevska This is ready for testing

Risk: High
Affected: App users with the Circle menu
Verify: Users can access all menu items as expected

Squash and merge: Enhance the accessibility of menu
